ee/packages/amplication-git-pull-service/src/git-pull-event/git-pull-event.controller.ts

```diff
@@ -9,6 +9,6 @@ export class GitPullEventController {
 
   @MessagePattern(KafkaTopics.GitExternalPush)
   async processRepositoryPushEvent(@Payload() message: any) {
-    await this.gitPullEventService.handlePushEvent(message.value);
+    await this.gitPullEventService.handlePushEvent(message);
```

Comment:
this change looks suspicious. Are you sure that contract was changed?

Reply:
https://docs.nestjs.com/migration-guide#kafka-messageevent-handlers